src/Components/Grid.php

```diff
@@ -37,9 +37,23 @@
  */
 class Grid extends Container {
 
+	const ATTR_MODE = 'mode';
+	const MODE_FIXEDWIDTH = 'fixedwidth';
+	const MODE_FLUID = 'fluid';
+
 	public function __construct( ChameleonTemplate $template, \DOMElement $domElement = null, $indent = 0 ) {
 
 		parent::__construct( $template, $domElement, $indent );
-		$this->addClasses( 'container' );
+		$containerClass = 'container';
+		if( $this->isFluidMode() ) {
+			$containerClass = 'container-fluid';
```

Comment:
Do we need `$containerClass`? Couldn't we just `$this->addClasses( 'container-fluid' )` here?